packages/data-service/src/legacy/legacy-connection-model.ts

```diff
@@ -155,13 +156,17 @@ export interface LegacyConnectionModel extends LegacyConnectionModelProperties {
 }
 
 export function convertConnectionModelToInfo(
-  model: LegacyConnectionModel
+  model: EJSON.SerializableTypes
```

Comment:
How does EJSON come into play here?

Could this be

```suggestion
  model: LegacyConnectionModelProperties
```

instead?

Reply:
My bad, changed it.